data/meson.build

```diff
@@ -0,0 +1,5 @@
+install_data(APPLICATION_ID + '.desktop',
+  install_dir: get_option('datadir') + '/applications')
+install_data('../Images/GPU_Viewer.png',
+  rename: APPLICATION_ID + '.png',
+  install_dir: get_option('datadir') + '/icons/hicolor/512x512/apps')
```

Comment:
These install_dir should use Meson's path joining functionality, `join_paths('a', 'b')` or since Meson 0.49.0, `'a' / 'b'`as syntactic sugar.

Do not use string concatenation: `'a' + '/b'`. It is far less readable than the syntactic sugar `/` operator, and it will normalize using string concatenation on paths which will eventually bite you in situations that include absolute paths.